seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/client/StarRocksStreamLoadVisitor.java

```diff
@@ -71,7 +71,7 @@ public Boolean doStreamLoad(StarRocksFlushTuple flushData) throws IOException {
         String host = getAvailableHost();
         if (null == host) {
             throw new StarRocksConnectorException(
-                    CommonErrorCodeDeprecated.ILLEGAL_ARGUMENT,
+                    CommonErrorCode.ILLEGAL_ARGUMENT,
```

Comment:
Please do not directly use common error code, we should create it in https://github.com/apache/seatunnel/blob/c1b2675ab0826bc90a16766b1811f9f3dc4c2c0c/seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonError.java#L58